.github/workflows/rust-test-with-style.yml

```diff
@@ -19,14 +19,16 @@ jobs:
       uses: actions-rs/toolchain@v1
       with:
         toolchain: stable
-    - name: Rust test
+        override: true
+    - name: Rust test with coverage
+      uses: actions-rs/tarpaulin@v0.1
       env:
         CC: ${{ matrix.compiler }}
         FC: gfortran-9
-      run: |
-        make info
-        make -j2
-        cargo test
-    - name: Rust style
+    - name: Codecov upload
+      uses: codecov/codecov-action@v1.0.2
+      with:
+        token: ${{secrets.CODECOV_TOKEN}}
```

Comment:
@jedbrown, when you get a chance, can you please put a `CODECOV_TOKEN` in the repo secrets?

Reply:
Should work now.